File: 1889-CheckIfNumberIsASumOfPowersOfThree/1889-CheckIfNumberIsASumOfPowersOfThree.java
// Last updated: 3/21/2026, 3:39:44 PM
class Solution {
    public boolean checkPowersOfThree(int n) {
        while(n>0){
            if((n%3)>1){
                return false;
            }
            n=(int)n/3;
            System.out.println(n);
        }
        return true;
    }
}

File: 1889-CheckIfNumberIsASumOfPowersOfThree/1889-CheckIfNumberIsASumOfPowersOfThree.py
# Last updated: 3/21/2026, 3:39:48 PM
class Solution:
    def checkPowersOfThree(self, n: int) -> bool:
        while n:
            if n%3 >1:
                return False
            n=n//3
            print(n)
        return True
        